### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by blankenbush**

### CODE REVIEW

File: src/UI/Areas/Identity/Pages/V4/Account/Manage/DeletePersonalData.cshtml.cs [hunk 1]
```diff
@@ -47,8 +47,7 @@ public class InputModel
 
         /// <summary>
         ///     This API supports the ASP.NET Core Identity default UI infrastructure and is not intended to be used
-        ///     directly from your code. This API may change or be removed in future releases.
-        /// </summary>
+        ///     directly from your code. This API may change or be removed in future releases.        /// </summary>
```
Comment (Eilon) [inline src/UI/Areas/Identity/Pages/V4/Account/Manage/DeletePersonalData.cshtml.cs]: Should revert this change.

Comment (blankenbush) [reply src/UI/Areas/Identity/Pages/V4/Account/Manage/DeletePersonalData.cshtml.cs]: Should be good now.

**Comment by Eilon** [comment]
@HaoK / @ajcvickers - the PR looks good to me.

**Comment by HaoK** [comment]
Thanks @blankenbush !
